jekyll/_cci2/server-3-operator-externalizing-services.adoc

```diff
@@ -70,11 +83,45 @@ Consider running at least two or more PostgreSQL replicas to enable recovery fro
 |===
 
 === Backing Up PostgreSQL
-PostgreSQL provides official documentation for backing up and restoring your PostgreSQL 12 install which can be found https://www.postgresql.org/docs/12/backup.html[here].
+PostgreSQL provides official documentation for backing up and restoring your PostgreSQL 12 install, which can be found https://www.postgresql.org/docs/12/backup.html[here].
 
 We strongly recommend the following:
 
 * Taking daily backups
 * Keeping at least 30 days of backup
 * Using encrypted storage for backups as databases might contain sensitive information
-* Performing a backup before each upgrade of CircleCI Server.
+* Performing a backup before each upgrade of CircleCI server.
+
+== MongoDB
+
+NOTE: If using your own MongoDB instance it needs to be version 3.6 or greater. 
+
+If you choose to use an external MongoDB instance, complete the following fields: 
+
+* *MongoDB connection host(s) or IP(s) (required)* - 
+The hostname or IP of your MongoDB instance. Specifying a port using a colon and multiple hosts for sharded instances are both supported.
+
+* *Use SSL for connection to MongoDB (required)* - 
+Whether to use SSL when connecting to your external MongoDB instance
+
+* *Allow insecure TLS connections (required)* - 
+If you use a self-signed certificate or one signed by a custom CA, you will need to enable this setting. However, this is an insecure setting and you should use a TLS certificate signed by a valid CA if you can.
+
+* *MongoDB user (required)* - 
+The username for the account to use. This account should have the dbAdmin role.
+
+* *MongoDB password (required)* - 
+The password for the account to use.
+
+* *MongoDB authentication source database (required)* - 
+The database that holds the account information, usually admin.
+
+* *MongoDB authentication mechanism (required)* - 
+The authentication mechanism to use, usually SCRAM-SHA-1.
+
+* *Additional connection options (optional)* - 
+Any other connection options you would like to use. This needs to be formatted as a query string (`key=value` pairs, separated by `&``, special characters need to be URL encoded). See the link:https://docs.mongodb.com/v3.6/reference/connection-string/[MongoDB docs] for available options.
+
+== Vault
+
+Complete the following Vault fields: 
```

Comment:
@nathanfishcircleci we need equivalents for Vault here I think

Reply:
Sorry!

Complete the following Vault fields: 

URL - The URL to your Vault service. 

Transit Path - Your Vault secrets transit path.

Token - The access token for vault.